ui/components/ChipGroup.tsx

```diff
@@ -19,7 +20,14 @@ function ChipGroup({ className, chips = [], onChipRemove, onClearAll }: Props) {
       {_.map(chips, (chip, index) => {
         return (
           <Flex key={index}>
-            <Chip label={chip} onDelete={() => onChipRemove([chip])} />
+            <Chip
+              label={
+                chip.slice(-filterSeparator.length) === `${filterSeparator}`
```

Comment:
Either do something like `chip.indexOf(filterSeparator) == chip.length - filterSeparator.length` (I probably have an off-by-one in there, I haven't tried it) so it 100% only manipulates empty chip values, or just drop this.

Reply:
@joshri why are you using a template literal

```
`${filterSeparator}`
```

here instead of just `filterSeparator`? Was the comparison not working?